### DISCUSSION TITLE
Profile and repository not found in search

### DISCUSSION THREAD

**Discussion by kdelint** [community/community#46426] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Neither my github profile (kdelint, created years ago) or my repository (IsogenicZ, created 6 days ago) can be found in github search, and are also not indexed by google it seems.

Edit: google searches now do find this discussion when searching for kdelint or IsogenicZ (within an hour of posting!) but still not my profile or the repository...

**Comment by SilasRodrigues19** [comment]
Hey @kdelint 

Your profile is already showing up, the repo still doesn't show up but I think it hasn't given enough time to be indexed there in searches.

![image](https://user-images.githubusercontent.com/47532703/216829407-d5f8b729-5d27-4e22-8a0d-fffdc647a7ba.png)
![image](https://user-images.githubusercontent.com/47532703/216829414-98b4b5db-e09e-4d70-b12b-dd8b1da4092d.png)

**Reply by SilasRodrigues19** [reply]
I saw that your repo is a fork, forks are not indexed in the fork user's search

**Comment by kdelint** [comment, marked as answer]
Thanks SilasRodrigues19! I hope Google will index my profile in the future, just to make this repo (and future ones) easy to find. If I google a colleagues username it does show up, with the available repositories (searching for github + repository name also works), that's all the visibility I would like :)...  

![image](https://user-images.githubusercontent.com/45465909/216830459-8ade7591-a77a-4d83-bc62-c61116bbedc3.png)

**Reply by SilasRodrigues19** [reply]
You can improve your repo  description, include many topics that make sense and make a very detailed `README.md`, this will help a lot in Google indexing.

I saw that your repo doesn't have any topics and the README file could have more details.

Here is a really cool example of a template for creating the README:

https://github.com/othneildrew/Best-README-Template

**Reply by kdelint** [reply]
Thanks for the suggestions, will implement when I have time! The day before yesterday google could find my profile, but today not anymore, this discussion is also sometimes found but then lost, probably indexing issues that could be Github or Google.
